File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.js
function solution(sizes) {
    let widths = [];
    let heights = [];
    
    sizes.forEach((size) => {
        widths.push(Math.max(...size));
        heights.push(Math.min(...size));
    })
    
    return Math.max(...widths) * Math.max(...heights);
}

File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.py
def solution(sizes):
    widths = []
    heights = []
    for w, h in sizes:
        long = max(w, h)
        short = min(w, h)
        widths.append(long)
        heights.append(short)
        
    print(widths)
    print(heights)
    return max(widths) * max(heights)